### ISSUE TITLE
Project doesn't seem to re-open correctly

### ISSUE THREAD

**Issue by stormont** [PancakeBot/PancakeCreator#61]
I saved a project, but when I re-open, I just see a blank canvas again. Looking at the file in a text editor, it seems like the data I saved is there.

Running on Mac OS X El Capitan.

**Comment by techninja** [comment]
I suspect this is an issue with layers/export bug. Post your file and we might be able to get a fix

**Comment by kennywhite** [comment]
Yeah file opening seems completely broken on macOS.  Not working on Catalina (10.15.1) either.
